tpm/tpm.go

```diff
@@ -872,3 +872,156 @@ func TakeOwnership(rw io.ReadWriter, newOwnerAuth digest, newSRKAuth digest, pub
 	raIn := []interface{}{ret, ordTakeOwnership, k}
 	return ra.verify(ca.NonceOdd, newOwnerAuth[:], raIn)
 }
+
+// CreateWrapKey creates a new RSA key for signatures inside the TPM. It is wrapped by the SRK (which is to say,
```

Comment:
nit: The style in this repo is to wrap comments to 80 columns.

Reply:
:+1: